rest/data/logkeeper.go

```diff
@@ -0,0 +1,16 @@
+package data
```

Comment:
I would defer to the app folks over organization but my intuition is this probably shouldn't live here since this won't be used by the rest routes at all.

Reply:
Is there a reason this needs to be in a package external to graphql at all? If it's only used by graphql then I would think this code could just live in the resolver itself, or the graphql/util.go file

Reply:
Sounds good, I wasn't sure if calling an external api from GraphQL was discouraged :o) glad to know it's not!

Reply:
actually idk maybe that's a good point, i wasn't thinking of this as an external thing 😅 we don't really have something similar that i can think of. It could live under model the same way cedar stuff seems to (https://github.com/evergreen-ci/evergreen/blob/0c54078f0a6112dd737ac88e048adcb608dd0079/model/task_log.go) but yeah idk, i guess the primary benefit of having it elsewhere would be if we wanted to use a mock version of the client for tests but it seems that wouldn't provide much value since we just return the results verbatim

Reply:
Yeah, I think for now interacting with Plank directly from the resolver is nice and lightweight, but maybe in the future this would make sense!